lib/phoenix/digester.ex

```diff
@@ -46,18 +47,48 @@ defmodule Phoenix.Digester do
     |> Enum.map(&(map_file(&1, input_path)))
   end
 
-  defp generate_manifest(files, output_path) do
+  defp load_digests(input_path) do
+    manifest_path = Path.join(input_path, "manifest.json")
+    if File.exists?(manifest_path) do
+      manifest_path
+      |> File.read!
+      |> Poison.decode!
+      |> Access.get("digests")
+    end
```

Comment:
I believe this needs to return `%{}` and never nil if I'm reading correctly. This means that it needs an else clause and it needs to account for the file format: `Access.get("digests", %{})`